src/openrct2/ride/vehicle.h

```diff
@@ -243,7 +243,7 @@ enum {
     VEHICLE_ENTRY_FLAG_RIDERS_SCREAM = 1 << 20,
     VEHICLE_ENTRY_FLAG_21 = 1 << 21,
     VEHICLE_ENTRY_FLAG_22 = 1 << 22,
-    VEHICLE_ENTRY_FLAG_23 = 1 << 23,
+    VEHICLE_ENTRY_FLAG_23 = 1 << 23,// oli414
```

Comment:
?

Reply:
Whoops that's one of my debug markers that I forgot to remove.

Reply:
I really have to better check my changes before I push. I'm still trying to find my workflow, sorry about that.